lib/lib-dynamodb/src/baseCommand/DynamoDBDocumentClientCommand.spec.ts

```diff
@@ -41,7 +41,8 @@ describe("DynamoDBDocumentClientCommand", () => {
       expect(options).toEqual({
         name: "DocumentMarshall",
         override: true,
-        step: "initialize",
+        relation: "before",
+        toMiddleware: "serializerMiddleware",
```

Comment:
this isn't strictly necessary since the serializer in in the serialize stage and this was in the initialize stage, but more explicit